http-server/server/server.go

```diff
@@ -10,25 +11,35 @@ import (
 	"time"
 )
 
-var pageAccessCount = 0
-var Param string
+// change this to use methods in the file
+type serverInfo struct {
+	pageAccessCount int
+	Param           string
+}
+
+// MakeServerInfo creates & returns newly initialized serverInfo structure with given args
+func MakeServerInfo(pageAccessCnt int, parameter string) *serverInfo {
```

Comment:
It kinda doesn't make a sense to set pageAccessCnt here via parameter.
Just set it init it in the constructor without param.